odoo/addons/test_search_panel_select_range/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "test search_panel_select_range",
```

Comment:
Why such a specific module name?
Why not simply `test_search_panel`?

Reply:
It would be shorter of course :-). If you agree, I will go for it.

Reply:
Concerning the localization of the addon, do you agree or should it be at the same level as test_mail, test_website?